src/DurableFunctions.TypedInterfaces/SourceGenerator/DurableFunctions.TypedInterfaces.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Thanks for review! Just checking: should I also update this project? and the below one src/WebJobs.Extensions.DurableTask.Analyzers/WebJobs.Extensions.DurableTask.Analyzers.csproj @cgillum

Reply:
Things that are `netstandard2.0` should be left as-is because there may be other compatibilities requirements that they have. But anything that's `netcoreapp3.1` should be upgrade to `net6.0`.